@here/harp-examples/src/geo-to-screen.ts

```diff
@@ -0,0 +1,286 @@
+  <br />     the red square is painted on the screen in screen coordinates, whereas the green cube
+  <br />     lives in the world.
+  <br />  Use the arrow keys or the gui to change the geoPosition
+  <br />  Jump to next worlds with "j" and "l"
+  `;
+        message.style.position = "absolute";
+        message.style.cssFloat = "right";
+        message.style.top = "10px";
+        message.style.left = "100px";
+        message.style.textAlign = "left";
+        message.style.textShadow = "0px 0px 2px gray";
+        document.body.appendChild(message);
+    }
+
+    function addScreenAnchorValues(): HTMLDivElement {
+        const screenAnchorOutput = document.createElement("div");
+        screenAnchorOutput.id = "screenAnchorOutput";
+        screenAnchorOutput.style.position = "absolute";
+        screenAnchorOutput.style.cssFloat = "bottom-right";
+        screenAnchorOutput.style.bottom = "10px";
+        screenAnchorOutput.style.left = "10px";
+        screenAnchorOutput.style.textAlign = "left";
+        screenAnchorOutput.style.textShadow = "0px 0px 2px gray";
+        document.body.appendChild(screenAnchorOutput);
+        return screenAnchorOutput;
+    }
+
+    window.addEventListener("keydown", event => {
+        switch (event.key) {
+            case "ArrowLeft":
+            case "ArrowRight":
+            case "ArrowUp":
+            case "ArrowDown":
+                startAnchorUpdate(event.key);
+                break;
+            default:
+                break;
+        }
+    });
+
+    let anchorUpdateTimeout: number;
+    function startAnchorUpdate(direction: string) {
+        const step = 1 / Math.pow(mapView.zoomLevel, 2);
+        switch (direction) {
+            case "ArrowLeft":
+                geoPosition.lng -= step;
+                break;
+            case "ArrowRight":
+                geoPosition.lng += step;
+                break;
+            case "ArrowUp":
+                geoPosition.lat += step;
+                break;
+            case "ArrowDown":
+                geoPosition.lat -= step;
+                break;
+            default:
+                stopAnchorUpdate();
+                break;
+        }
+
+        anchorUpdateTimeout = setTimeout(updateAnchors, 0.1) as any;
+    }
+
+    function stopAnchorUpdate() {
+        if (anchorUpdateTimeout !== undefined || anchorUpdateTimeout !== 0) {
+            clearTimeout(anchorUpdateTimeout);
+        }
+    }
+
+    window.addEventListener("keyup", event => {
+        switch (event.key) {
+            case "ArrowLeft":
+            case "ArrowRight":
+            case "ArrowUp":
+            case "ArrowDown":
+                stopAnchorUpdate();
+                break;
+            case "j":
+                mapView.lookAt({
+                    target: new GeoCoordinates(
+                        mapView.target.latitude,
+                        (mapView.target.longitude -= 360)
```

Comment:
This is somehow not reflected in the GUI. I always see normalized longitude there

Reply:
The thing is, what you see in the GUI is the geoPosition that is used for the getScreenPosition call, ... this moves the mapView target

Reply:
ah, missed that. We don't move the object into the other world but we move all the worlds instead :smile: